carbonmark/lib/types/carbonmark.types.ts

```diff
@@ -28,6 +28,8 @@ export type CategoryName = keyof typeof CATEGORY_INFO;
 export interface DigitalCarbonCredit {
   id: string;
   vintage: number;
+  tokenAddress: string;
+  tokenId: string | null;
```

Comment:
The sooner we integrate with the new retire endpoints and the sooenr we get rid of these types the better.

Do you have a ticket for integrating with the retire endpoints?

Reply:
https://github.com/KlimaDAO/klimadao/issues/2128